cou/commands.py

```diff
@@ -209,6 +223,18 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         type=purge_before_arg,
         required=False,
     )
+    subcommand_common_opts_parser.add_argument(
+        "--skip-apps",
+        dest="skip_apps",
+        choices=["vault"],
+        action=SkipAppsArgumentAction,
+        nargs="*",
```

Comment:
```suggestion
        nargs="+",
        default=[],
```

With nargs `*`, it's possible to pass `--skip-apps` with no arguments, which doesn't make much sense.  

Also, adding a default to empty list means that we'll always have a list as the value for args.skip_apps.  This means we can also drop the `skip_apps or set()` logic.

Reply:
> With nargs *, it's possible to pass --skip-apps with no arguments, which doesn't make much sense.

Yeah, I saw that too, that's why there's `skip_apps or set()`.